common/views/themes/base/normalize.ts

```diff
@@ -1,5 +1,5 @@
 export const normalize = `
-/*! normalize.css v5.0.0 | MIT License | github.com/necolas/normalize.css */
```

Comment:
didn't mean to put this change in this pr, but unless anyone wants me to move it into a separate one I think I'll leave it here instead of opening a separate pr and reverting this change at this point.